cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -238,8 +250,6 @@ def load_resource_file(
                 "authentication",
                 "sourceOidcCredentials",
                 "destinationOidcCredentials",
```

Comment:
```suggestion
```
since L263-L271 are going to raise errors they shouldn't be handled here either?